tools/kdc.json

```diff
@@ -29,7 +29,8 @@
   },
   "networks": [
     {
-      "mode": "container/bridge"
+      "name": "dcos",
```

Comment:
Nice! Does this mean that it works on non-overlay networks too?

Reply:
Managed to make it simply work on `host`.